app/views/partials/foyer/individu-form.html

```diff
@@ -184,21 +184,24 @@
     </div>
   </div>
 
-  <div class="form-group from-group-lg" ng-if="captureTauxInvalidite()">
+  <div class="form-group from-group-lg" ng-if="capturetauxIncapacite()">
     <label class="control-label col-sm-4" for="eligibiliteAss">
-      Quel est {{ isDemandeur() ? 'votre' : 'son' }} taux d'invalidité ?
+      Quel est {{ isDemandeur() ? 'votre' : 'son' }} taux d'incapacité ?
+      <div class="small">
```

Comment:
`help-block` looks better, but not consistent with the other text.
We could set all to `help-block` here.
`small`is otherwise used in situation recap